### ISSUE TITLE
.NET Isolated DateTime Route parameter issue

### ISSUE THREAD

**Issue by rodchenkov** [Azure/azure-functions-dotnet-worker#851]
Hi and looks like it is same issue as this one #521 but for DateTime.

```
[Function("Rates")]
public async Task<string> Run(
    [HttpTrigger(AuthorizationLevel.Function, "get", Route = "rates/{onDate:datetime}")] HttpRequestData req,
    DateTime onDate,
    FunctionContext executionContext)
{
  ...
}
```

The error that I'm getting

> Exception: Microsoft.Azure.Functions.Worker.Diagnostics.Exceptions.FunctionInputConverterException: Error converting 1 input parameters for Function 'Rates': Cannot convert input parameter 'onDate' to type 'System.DateTime' from type 'System.String'.

Thanks

**Comment by kshyju** [comment]
@rodchenkov This is fixed in [1.8.0-preview3](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker/1.8.0-preview3). Could you give it a shot? Thanks!

**Comment by rodchenkov** [comment]
Hi @kshyju and thank you, now all is working.
